_posts/minutes/2016-09-16-minutes.markdown

```diff
@@ -0,0 +1,93 @@
+#### BAKERSFIELD COLLEGE
+
+#### Computer Science Club
+
+#### 1801 Panorama Drive, Business Building, Room 5
+
+#### Bakersfield, California 93305
```

Comment:
There is much that will be the same each time, I'm just a little shaky on how to make a base template in github. Open to learning though.